### PR TITLE
new package: psalg

### PR DESCRIPTION
**Pull request by eugeneswalker**
FYI @valmar

### CODE REVIEW

**Comment by tldahlgren** [comment]
Updated branch to pick up the newly merged `xtcdata` package.

File: var/spack/repos/builtin/packages/psalg/package.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+from spack.package import *
+
+
+class Psalg(CMakePackage):
+    """LCLS II Developement: PSAlg."""
+
+    homepage = "https://github.com/slac-lcls/lcls2"
+    url = "https://github.com/slac-lcls/lcls2/archive/refs/tags/3.3.37.tar.gz"
+
+    maintainers = ["valmar"]
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/psalg/package.py]: @valmar Are you okay with being listed as a maintainer on this package?  See https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers for more information.

Comment (tldahlgren) [inline var/spack/repos/builtin/packages/psalg/package.py]: Please use the new syntax for listing maintainers:

```suggestion
    maintainers("valmar")
```

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Confirmed the sha256.  Need confirmation on the maintainer and requested the new maintainers syntax be used.

**Comment by tldahlgren** [comment]
`spack spec` succeeded.

**Comment by eugeneswalker** [comment]
@spackbot run pipeline

File: var/spack/repos/builtin/packages/psalg/package.py (hunk 1 above)
Comment (eugeneswalker) [reply var/spack/repos/builtin/packages/psalg/package.py]: @valmar and I have corresponded by email and it is OK.

Comment (valmar) [reply var/spack/repos/builtin/packages/psalg/package.py]: I apologize for the delay in reacting to this. I am OK being listed as maintainer of psalg

**Comment by valmar** [comment]
The PR looks OK to me!

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**